test/extended/util/ibmcloud/provider.go

```diff
@@ -25,31 +18,3 @@ func newProvider() (framework.ProviderInterface, error) {
 type Provider struct {
 	framework.NullProvider
 }
-
-// TODO: Determine if this can be avoided by running worker kubelets without
-// --enable-controller-attach-detach=false
-// e2e: https://bugzilla.redhat.com/show_bug.cgi?id=1825034 - Mock CSI tests fail on IBM ROKS clusters
-func (p *Provider) FrameworkBeforeEach(f *framework.Framework) {
-	_, err := f.ClientSet.RbacV1().ClusterRoleBindings().Get(context.Background(), "e2e-node-attacher", metav1.GetOptions{})
-	if errors.IsNotFound(err) {
```

Comment:
the error reported in the BZ was an "already exists" error.  If this "IsNotFound" check is working properly, the code should not attempt to create the CRB if it already exists, so do we know why this check wasn't sufficient?